src/components/authpage/LoginPage.jsx

```diff
@@ -0,0 +1,147 @@
+                        <label htmlFor="email" className="input-label fw-600">
+                            Email:
+                        </label>
+                        <input
+                            id="email"
+                            type="email"
+                            value={email}
+                            className={
+                                "input-field" +
+                                (emailError ? " input-color-error" : " ")
+                            }
+                            onChange={(e) => setEmail(e.target.value)}
+                        />
+                        {emailError && (
+                            <span className="input-message">
+                                Please enter valid email
+                            </span>
+                        )}
+                    </div>
+                    <div className="input-wrapper p-y-1 w-100">
+                        <label
+                            htmlFor="password"
+                            className="input-label fw-600"
+                        >
+                            Password:
+                        </label>
+                        <input
+                            id="password"
+                            type={showPassword ? "text" : "password"}
+                            value={password}
+                            className={
+                                "input-field" +
+                                (passwordError ? " input-color-error" : " ")
+                            }
+                            onChange={(e) => setPassword(e.target.value)}
+                        />
+                        {passwordError && (
+                            <span className="input-message">
+                                Please enter password more that 8 characters
+                            </span>
+                        )}
+                    </div>
+                    <div className="flex justify-content-between w-100 p-y-1">
+                        <label className="font-small fw-500 flex align-items-center">
+                            <input
+                                type="checkbox"
+                                name="show-password"
+                                checked={showPassword}
+                                onChange={() => setShowPassword((s) => !s)}
+                            />{" "}
+                            <span className="p-0_5"> </span> Show Password
+                        </label>
+                        <button
+                            style={guestCredentialsStyle}
+                            className="btn-link-primary button-link fw-500"
+                            onClick={useGuestCredentials}
+                        >
+                            Use Test Credentials
+                        </button>
+                    </div>
+                    <div className="p-1"></div>
```

Comment:
What is this extra div here for?

Reply:
It handles padding basically so I have utility classes as well so thought of using it this way.